.github/workflows/VersionCalPRComment.yml

```diff
@@ -0,0 +1,131 @@
+      - name: Display Diff Modules
+        run: |
+            for mod in `cat changed_modules`
+            do
+              echo changed module: ${mod}
+            done
+      - uses: actions/checkout@master
+        with:
+           repository: Azure/azure-cli
+           path: ./azure-cli
+      - name: Show workdirectory after cli cloned
+        run: |
+           pwd
+           ls
+      - name: Organize parent workdirectory
+        run: |
+           mv azure-cli ../
+           cd ../
+           pwd
+           ls
+      - name: Install azdev
+        run: | 
+           python -m pip install --upgrade pip
+           set -ev
+           python -m venv env
+           chmod +x env/bin/activate
+           source ./env/bin/activate
+           pip install azdev
+           azdev --version
+           cd ../
+           azdev setup -c azure-cli -r azure-cli-extensions --debug
+           az --version
+           pip list -v
+      - name: Gen Base and Diff Metadata
+        id: get_comment_message
+        env:
+          pr_label_list: ${{ toJson(github.event.pull_request.labels.*.name) }}
+          base_meta_path: "./base_meta/"
+          diff_meta_path: "./diff_meta/"
+          output_file: "version_update.txt"
+        run: |
+           chmod +x env/bin/activate
+           source ./env/bin/activate
+           echo GITHUB_BASE_REF: ${GITHUB_BASE_REF}
+           git checkout ${GITHUB_BASE_REF}
+           for mod in `cat changed_modules`
+           do
+              echo **************************
+              echo changed module: ${mod}
+              azdev extension add ${mod}
+              azdev command-change meta-export ${mod} --meta-output-path ./${base_meta_path}/
+              azdev extension remove ${mod}
+              echo **************************
+              echo -e "\n\n\n\n\n"
+           done
+           echo GITHUB_HEAD_REF: ${GITHUB_HEAD_REF}
+           git checkout ${GITHUB_HEAD_REF}
+           for mod in `cat changed_modules`
+           do
+              echo **************************
+              echo changed module: ${mod}
+              azdev extension add ${mod}
+              azdev command-change meta-export ${mod} --meta-output-path ./${diff_meta_path}/
+              azdev extension remove ${mod}
+              echo **************************
+              echo -e "\n\n\n\n\n"
+           done
+           echo ls base_meta_path:
+           ls ./${base_meta_path}/
+           echo ls diff_meta_path
+           ls ./${diff_meta_path}/
+           python scripts/ci/release_version_cal.py
+           # echo "commit_message=$(cat ${output_file})" >> $GITHUB_ENV
+      - name: comment on the pull request
+        uses: mshick/add-pr-comment@v2
+        with:
+           message-path: |
+            version_update.txt
+           message-failure: |
```

Comment:
What are the trigger conditions for message-failure?

Reply:
In case release_version_cal.py runs with unexpected errors and out file is not generated, use this hint for users to manage the version number manually.